test_requests.py

```diff
@@ -411,6 +416,21 @@ def test_DIGESTAUTH_QUOTES_QOP_VALUE(self):
         r = requests.get(url, auth=auth)
         assert '"auth"' in r.request.headers['Authorization']
 
+    def test_DIGESTAUTH_THREADED(self):
```

Comment:
Separately, while this test does work, it's both overly aggressive and doesn't necessarily reliably fail. Is there a way of us testing this without requiring us to make 100 HTTP requests to do it?